### ISSUE TITLE
The hint of semantic widget won't voice out when focusing on it at the first time (at least on iOS 13.3.1 and 13.5.1)

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#60437]
b/159686327

Observe that when using Semantics widget in Flutter on iOS platform, the hint wont voice out when focusing on it at the first time, but will voice out if swipe right and swipe back.

Create Semantics widget as below.

```dart
Semantics(
  label: 'label',
  hint: 'hint',
  excludeSemantics: true,
  child: ...,
```

Turn on VoiceOver, and swipe to focus on the widget:

- On iOS 13.3.1 & iOS 13.5.1, it won't voice out at the first time.
- On iOS 13.3, it will voice out as normal (including the first time to focus on it).

**Comment by nscobie** [comment]
_(**Unable to reproduce**, following up with the customer internally. Asking for any corrections to my repro app, and which Flutter version they're on.)_

My setup:
- Device: `iPhone 8` running `iOS 13.5.1`
- Flutter: `1.20.0-3.0.pre.68`

I have tried both having the Semantics node in question being the initially focused widget when launching the app (commenting out `Text('pre')` below), and having it be the second widget in the tree and swiping right to focus it. In both cases it reads _**"label... hint"**_ without fail, albeit with a significant delay before "hint".

**Attempted repro app:**
```dart
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}

// Trying to repro https://github.com/flutter/flutter/issues/60437
class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
        body: ListView(
          children: [
            Text('pre'),
            Semantics(
              label: 'label',
              hint: 'hint',
              excludeSemantics: true,
              child: Text('child'),
            ),
            Text('post'),
          ],
        ),
      ),
    );
  }
}

```

**Comment by mehmetf** [comment]
Closing as cannot reproduce.
